db/migrate/20151112110911_add_event_id_to_event_logs.rb

```diff
@@ -0,0 +1,6 @@
+class AddEventIdToEventLogs < ActiveRecord::Migration
+  def change
+    add_column :event_logs, :event_id, :integer
+    change_column :event_logs, :event, :string, default: ''
```

Comment:
Would be better to not add the default here, and instead make the code carry on filling the `event` column with the event's description in order to maintain backwards compatibility.

The ADD COLUMN will still block, but should only be for a few seconds(?). Needs testing on staging, but an alternative solution might be to use https://www.percona.com/doc/percona-toolkit/2.1/pt-online-schema-change.html